python/tvm/meta_schedule/tune.py

```diff
@@ -17,7 +17,8 @@
 """User-facing Tuning API"""
 # pylint: disable=import-outside-toplevel
 import logging
-import os.path
+from pathlib import Path
```

Comment:
Pathlib is definitely a good library, but perhaps it's less plausible if we mix `os.path` and `Pathlib`. In our particular case, shall we just do:

```python
os.makedirs(path, exist_ok=False)
```

Reply:
I think it might be acceptable to use `exist_ok = True` in this case cause the logs folder could be reused.